server/models/Profile.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+
+const availabileTimes = {
+  startTime: Number,
+  endTime: Number,
+},
```

Comment:
So, we're going to keep availability pretty basic. Just monday to friday between X and Y. So we'll need to think on implementing this a bit more structured in the future. But otherwise somewhat on the right track 👍🏻